cli/flox/src/commands/environment.rs

```diff
@@ -433,7 +434,7 @@ impl Activate {
                 })),
             ))) => {
                 let mut message = format!(
-                    "This environment is not yet compatible with your system ({system}).",
+                    "Environment not yet compatible with your system ({system}).",
```

Comment:
blocking: I don't think this specific change is requested in the ticket, and I believe it's contrary to our style guidelines (use complete sentences)

Reply:
Fixed